docs/getting-started/launching-one-node/docker.md

```diff
@@ -9,51 +9,79 @@ title: Orion Docker
 
-### Build
+## 2) Orion Docker Image
 
-To build the docker image, the following steps need to be executed:
+Orion node can be launched by either
+   1. Pulling the published docker image in the docker hub (or)
+   2. Building the image from scratch
 
-  1. To clone the repository, first, create required directories using the command `mkdir -p github.com/hyperledger-labs`
-  2. Change the current working directory to the above created folder by issing the command `cd github.com/hyperledger-labs`
-  3. Clone this repository with `git clone https://github.com/hyperledger-labs/orion-server`
-  4. Change the current working directory to the repository root directory by issuing `cd orion-server`
-  5. Optional. Generate all cryptographic materials. For more info see [crypto-materials](crypto-materials).
-  6. To build the docker image, issue the command `make docker` which will create a docker image named `orion-server`
-  7. Optional. Versions of docker image stored once in while in docker hub docker repository as `orionbcdb/orion-server`
+Next, we explain the both options.
 
-### Start Orion in a Docker Container
-If you only plan to try Orion, you can use the pre-existing configuration and crypto materials stored in folder `deployment`.
+### 2.1) Pull Orion Docker Image from Docker Hub
 
-To do that, you just have to run the following:
+Orion docker image is available in docker hub repository. To pull the image, execute the following command:
+```
+docker pull orionbcdb/orion-server
 ```
-docker run -it --rm -v $(pwd)/deployment/crypto/:/etc/orion-server/crypto -p 6001:6001 -p 7050:7050 orion-server
-``` 
-This command will store all the database data inside the running container, so that container termination will remove all data.
 
-If you plan more that just trying Orion, first you have to create __new__ and __unique__ cryptographic materials and optional, update Orion server configuration.   
+### 2.2) Build Your Own Orion Docker Image
 
-To create a minimal set of cryptographic materials, execute the `cryptoGen.sh` script as shown below.
-```
-./scripts/cryptoGen.sh deployment
+If you would like to build your own image rather than pulling it from the docker hub, execute the following steps:
+
+  1. Create the required directory using the command `mkdir -p github.com/hyperledger-labs`
+  2. Change the current working directory to the above created directory by issing the command `cd github.com/hyperledger-labs`
+  3. Clone the server repository with `git clone https://github.com/hyperledger-labs/orion-server`
+  4. Change the current working directory to the repository root directory by issuing `cd orion-server`
+  5. To build the docker image, issue the command `make docker` which will create a docker image named `orionbcdb/orion-server`
+
+## 3) Start Orion in a Docker Container
+
+To start a node, we need a certificate authority and crypto materials for the node and admin users. To simplify this task, we have provided sample
```

Comment:
for the node and admin users -> for the server node and the admin user.

Reply:
Done